features/netsocket/TCPSocket.cpp

```diff
@@ -89,11 +100,13 @@ nsapi_error_t TCPSocket::connect(const SocketAddress &address)
 
     /* Non-blocking connect gives "EISCONN" once done - convert to OK for blocking mode if we became connected during this call */
     if (ret == NSAPI_ERROR_IS_CONNECTED && blocking_connect_in_progress) {
+    _socket_stats.stats_update_socket_state(this, SOCK_CONNECTED);
```

Comment:
```suggestion
        _socket_stats.stats_update_socket_state(this, SOCK_CONNECTED);
```

Reply:
Its already fixed.. travis CI was not getting triggered automatically, hence error is still present